src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java

```diff
@@ -155,6 +155,32 @@ public void testCreate() throws Exception {
         testCreationModificationMessage(createdModificationWithOnlyMetadata);
     }
 
+    @Test
```

Comment:
in my opinion :
When creating a modification, it is set to active by default, so there’s no need to explicitly approve the opposite.